plugins/janus_videoroom.c

```diff
@@ -785,6 +785,19 @@ room-<unique room ID>: {
 		// Other publishers
 	]
 }
+\endverbatim *
+ *
+ * To enable or disable recording while the conference is in progress,
```

Comment:
This should clarify that it's a way to start/stop recording all participants in a bulk. We have ways to enable/disable recordings individually for different participants (which is still the preferred way for us since it allows us to use different paths, and what we use in production), and the sentence above creates ambiguity.

Reply:
Agree, updating.